content/en/docs/setup/best-practices/node-conformance.md

```diff
@@ -19,7 +19,7 @@ To run node conformance test, a node must satisfy the same prerequisites as a
 standard Kubernetes node. At a minimum, the node should have the following
 daemons installed:
 
-* Container Runtime (Docker)
+* Container Runtime (CRI-compatible container runtime example Docker, Containerd,CRI-O etc.)
```

Comment:
```suggestion
* CRI-compatible container runtimes such as Docker, containerd, and CRI-O
```
To be more concise

Reply:
> To be more concise

I also think this would be good and concise So commiting this changes.